net-im/libcommuni/metadata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		A cross-platform IRC framework written with Qt.
+		Communi provides a set of tools for enabling
+		IRC connectivity in Qt-based C++ and QML applications.
+	</longdescription>
+	<use>
+		<flag name="benchmarks">Install optional benchmarks.</flag>
+		<flag name="debug">Build as a debug release.</flag>
+		<flag name="examples">Install optional examples.</flag>
+		<flag name="icu">Use dev-libs/icu instead of app-i18n/uchardet.</flag>
```

Comment:
'instead of' does not really fit here since you still need to enable the other flag.

Reply:
I will leave it here. Instead, I will drop uchardet as a use flag. Uchardet is anyway the default, used by libcommuni. So anyone, who sets now icu, will be use this instead of uchardet.

Reply:
Well, I would actually suggest doing the opposite. Given that uchardet is preferred, `IUSE=+uchardet` is cleaner in the intent, while `IUSE=icu` may be accidentally enabled via over-eager global USE flags.

Reply:
Hm, yeah, makes more sense. I changed the logic now.